src/app/auth/pending-verification/pending-verification.page.ts

```
@@ -2,50 +2,70 @@
+    });
   }
 
-  resendVerificationLink(email: string) {
+  resendVerificationLink(email: string): void {
     this.isLoading = true;
-    const orgId = this.activatedRoute.snapshot.params.orgId;
+    const orgId = this.activatedRoute.snapshot.params.orgId as string;
 
     this.routerAuthService
       .resendVerificationLink(email, orgId)
       .pipe(tap(() => (this.isLoading = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)